README.md

```diff
@@ -1,30 +1,31 @@
-dnx-watch
+dotnet-watch
 ===
-`dnx-watch` is a file watcher for `dnx` that restarts the specified application when changes in the source code are detected. Its' purpose is to replace `dnx --watch`.
+`dotnet-watch` is a file watcher for `dotne` that restarts the specified application when changes in the source code are detected.
 
 ### How To Install
-From a console window run the following command `dnu commands install Microsoft.Dnx.Watcher` 
-Then the `dnx-watch` command will become available.
 
-To install the latest unstable release add the following parameter to the command `--fallbacksource https://myget.org/F/aspnetvnext/api/v3/index.json`
+Add `dotnet-watch` to the `tools` section of your `project.json` file:
 
-### How To Use
-`dnx-watch <arguments>`
-
-Example:
-
-* To run the command `kestrel` in the current folder: `dnx-watch kestrel`
-* To run the command kestrel in a different folder: `dnx-watch --project C:\myproject --dnx-args kestrel`
-* To run the command kestrel in a different folder with extra arguments: `dnx-watch --project C:\myproject --dnx-args kestrel arg1 arg2`
+```
+{
+...
+  "tools": {
+    "dotnet-watch": "1.0.0-*"
+  }
+...
+}
+```
 
-AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio3aj/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/dnx-watch/branch/dev)
-
-Travis:   [![Travis](https://travis-ci.org/aspnet/dnx-watch.svg?branch=dev)](https://travis-ci.org/aspnet/dnx-watch)
+### How To Use
+- `dotnet watch` (runs the application without arguments)
+- `dotnet watch foo bar` (runs the application with the arguments `foo bar`)
+- `dotnet watch --exit-on-change -- foo bar` (runs the application with the arguments `foo bar`. In addition it passes `--exit-on-change` to the watcher).
+- `dotnet watch --exit-on-change -- --exit-on-change` (runs the application with the argument `--exit-on-change`. In addition it passes `--exit-on-change` to the watcher).
```

Comment:
This example seems very confusing. How is it different from the previous example?